### ISSUE TITLE
Loading KeywordIO in OMEdit gives syntax warning

### ISSUE THREAD

**Issue by dietmarw** [christiankral/KeyWordIO#9]
When loading KeywordIO I get the following warning from OMEdit
```
[1] 10:21:19 Syntax Warning
[KeyWordIO/KeyWordIO/readLineWithoutCache.mo: 11:24-11:1688]: Lexer treating \ as \\, since \0 is not a valid Modelica escape sequence.
```
Not sure if this is a false positive.

**Comment by dietmarw** [comment]
It also **looks** like that warning is keeping OMEdit from loading the library correctly. Since now only the Icon and documentation view can be activated whilst the rest stay deactivated. Never seen that before.
![image](https://user-images.githubusercontent.com/9332/77895960-cf21e080-7277-11ea-96e7-89407ad3cd57.png)

Dymola does not complain but also non of the file reading seems to work.

**Comment by christiankral** [comment]
Issue is resolved by 8fac47d33cc62bd7df941c420d56a2bd40c9302d
